Automattic/jetpack
# Backup: Add backup file browser reference in backup admin page

This PR aims to reorganize a bit the information we present on the backup admin dashboard to reduce redundancy and include the `Manage your backup files` section to start promoting the backup file browser in the backup plugin.

## Proposed changes:
* Remove `Your site's heartbeat` section
* Move `Restore points created with every edit` to the top
* Add new learn more segment `Manage your backup files` pointing to the [file browser introduction blog post](https://jetpack.com/blog/introducing-backup-file-browser/), with a `jetpack_backup_learn_file_browser_click` new Tracks event.
* Rename `Or view your most recent restore point` to `View activity log`.
* Rename `Learn more` under the real-time section to `Learn about real-time backups`.

| Before | After |
|---|---|
| ![image](https://github.com/Automattic/jetpack/assets/1488641/e424fc45-642d-42fe-9e30-64a9cb42a45e) | ![image](https://github.com/Automattic/jetpack/assets/1488641/550a2c15-5ee2-4f92-9585-6c3fce2d4267) |

### Other information:


## Jetpack product discussion


p1HpG7-o4Y-p2

## Does this pull request change what data or activity we track or use?


We are now tracking a new `Learn more` link.

## Testing instructions:





* Spin up a Jurassic Ninja site with just the Jetpack VaultPress Backup plugin using this branch. You could use this link: https://jurassic.ninja/create?jetpack-beta&branches.jetpack-backup=update/backup-admin-information&wp-debug-log&nojetpack
* Connect the site to Jetpack and add a Jetpack VaultPress Backup plan.
* Navigate to VaultPress Backup page.
* Validate you see all the elements as the `After` screenshot shared above and ensure the links point to the expected destination.

Related to https://github.com/Automattic/jetpack-roadmap/issues/179
